content/en/docs/collector/deployment/best-practices.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Best Practices
+description: How to configure collectors for various use cases
+weight: 4
+---
+
+Now that you are equipped with the essential deployment patterns for the
+collector, let's have a closer look at best practices for collector (pipeline)
+configurations for different use cases.
+
+## Fan Out
+
+Export signals into more than one backend. For example, for testing, policy
+(signals from dev/test go into backend X, prod goes into Y), or migrations from
+one back-end to another (cut-over) or different signal types go into different
+backends.
+
+<!-- prettier-ignore-start -->
+{{< ot-tabs "Per type" "Per env" "Migration" >}}
+{{< ot-tab lang="yaml">}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  jaeger:
+    endpoint: "https://jaeger.example.com:14250"
+    insecure: true
+  prometheusremotewrite:
+    endpoint: "https://prw.example.com/v1/api/remote_write"
+
+service:
+  pipelines:
+    metrics/prod:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheusremotewrite]
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger]
+{{< /ot-tab >}}
+
+{{< ot-tab lang="yaml">}}
+receivers:
+  otlp:
+    protocols:
+      grpc:
```

Comment:
Using this configuration will produce a warning about DoS attacks (see [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks)). We should set the endpoint explicitly